whylog/tests/test_regexassistant/__init__.py

```diff
@@ -0,0 +1,74 @@
+from unittest import TestCase
+from nose.plugins.skip import SkipTest
+from whylog.regexassistant import RegexAssistant
+from whylog.teacher import Interval
+
+
+class FrontInput(object):
+    """
+    FrontInput for test purpose only while module Front is not implemented yet.
+    """
+
+    def __init__(self, offset, line_content, resource_location):
+        self.offset = offset
+        self.line_content = line_content
+        self.resource_location = resource_location
+
+
+class TestBasic(TestCase):
+    def test_one(self):
+        content1 = "2015-12-03 12:08:09 Connection error occurred on beta36"
+        content2 = "2015-12-03 12:10:10 Data migration from beta36 to beta21 failed"
+        content3 = "2015-12-03 12:11:00 Data is missing on beta21"
+
+        cause1 = FrontInput(903, content1, None)
+        cause2 = FrontInput(53, content2, None)
+        effect = FrontInput(1087, content3, None)
+
+        regex_assistant = RegexAssistant([cause1, effect])
```

Comment:
calling `RegexAssistant` like that suggests that it doesn't need to know which line is cause and which is the effect, which, I expect, will create problems in the future. We should force the GUI caller to tell `RegexAssistant` which line is the effect.